.github/workflows/edge_ghpage.yml

```
@@ -0,0 +1,29 @@
+name: Build and Deploy Edge version to GH Pages
+on:
+ workflow_dispatch:
+ push:
```

Your file does not pass `yarn prettier`.
These changes are result of `yarn fix`.

```suggestion
  workflow_dispatch:
  push:
```